chains/ethereum/transactions/addAgreement.js

```diff
@@ -0,0 +1,23 @@
+import { ethers } from "ethers"
+
+export const addAgreement = async (details) => {
+  try {
+    const paymentAmountInWei = ethers.utils.parseUnits(details.paymentAmount.toString(), 18);
+
+    const tx = await details.contract.createAgreement(
+      details.title,
+      details.description,
+      details.professional,
+      details.skills,
```

Comment:
we missed the information about which community this agreement belongs to. We should design this, also in the contract (have a community registry, etc)

Reply:
Noted, will add this feature to the Solana program.